src/pybind/mgr/mgr_module.py

```diff
@@ -311,11 +311,15 @@ def call(self, mgr, cmd_dict, inbuf):
         for a, d in self.args_dict.items():
             if 'req' in d and d['req'] == "false" and a not in cmd_dict:
                 continue
-            if kwargs_switch or (isinstance(cmd_dict[a], str) and '=' in cmd_dict[a]):
-                mgr.log.debug('found kwarg, assuming all following args are kw style')
-                kwargs_switch = True
+
+            if isinstance(cmd_dict[a], str) and '=' in cmd_dict[a]:
+                k, arg = cmd_dict[a].split('=', 1)
+                if k in self.args_dict:
+                    kwargs_switch = True
+
+            if kwargs_switch:
                 try:
-                    k, arg = cmd_dict[a].split('=')
+                    k, arg = cmd_dict[a].split('=', 1)
```

Comment:
@sebastian-philipp @jan--f please note, this expression does not throw if `=` does not exist in `cmd_dict[a]`.

Reply:
The value unpacking does throw if split returns only a single split.